### PR TITLE
15360 us forms system when to use

### PR DESCRIPTION
**Pull request by rianfowler**
assumes mermaid support is incoming: 

screen shot 2019-03-07 at 12 36 06

screen shot 2019-03-07 at 11 55 11

### CODE REVIEW

File: packages/documentation/src/pages/when-to-use-usfs.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: When to use the US Forms System
+---
+
+## General structure of applications built with US Forms System
+
+The US Forms System provides a simple way to create a consistent form experience for veterans. Applications created with this framework follow this flow:
```
Comment (rianfowler) [inline packages/documentation/src/pages/when-to-use-usfs.mdx]: Not really sure how this document fits into the overall flow - we could expand the explanation to include more overview info or include this in another overview doc.

File: packages/documentation/src/pages/when-to-use-usfs.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: When to use the US Forms System
```
Comment (rianfowler) [inline packages/documentation/src/pages/when-to-use-usfs.mdx]: Is it appropriate to refer to this as the US Forms System?

File: packages/documentation/src/pages/when-to-use-usfs.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (jbalboni) [inline packages/documentation/src/pages/when-to-use-usfs.mdx]: I think this file could go in the forms folder.

File: packages/documentation/src/pages/when-to-use-usfs.mdx (hunk 2 above)
Comment (jbalboni) [reply packages/documentation/src/pages/when-to-use-usfs.mdx]: That's a good question. Maybe just "forms system"?

File: packages/documentation/src/pages/when-to-use-usfs.mdx (hunk 1 above)
Comment (jbalboni) [reply packages/documentation/src/pages/when-to-use-usfs.mdx]: I think this probably would fit into an overview/introduction page.

File: packages/documentation/src/sidebar.js [hunk 4]
```diff
@@ -74,6 +74,7 @@ module.exports = {
       items: [
         {
           name: 'Introduction',
+          href: 'forms/when-to-use-fs'
```
Comment (rianfowler) [inline packages/documentation/src/sidebar.js]: Just going to put this here for now. we can expand this document or reorganize it during an edit phase.

**Pull request merged**